core/base/src/model.ts

```diff
@@ -1,6 +1,7 @@
 import { isPlainObject } from "is-plain-object";
 import { validate } from "@kubernetes-models/validate";
 import { TypeMeta } from "./meta";
+import { KubernetesObject, KubernetesObjectWithSpec, V1ObjectMeta } from '@kubernetes/client-node';
```

Comment:
Maybe we don't have to import the types from `@kubernetes/client-node`. Instead, we can just copy the types we want into the base package?